utils/flakiness-dashboard/processing/dashboard_v2.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {SimpleBlob, flattenSpecs} = require('./utils.js');
+
+const DASHBOARD_VERSION = 1;
```

Comment:
should this not be 2?
```suggestion
const DASHBOARD_VERSION = 2;
```

Reply:
v2 dashboard is side-by-side with the v1 dashboard in a separate blob - so no changes are necessary